deploy/02-rbac.yaml

```diff
@@ -1,5 +1,5 @@
-kind: Role
-apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
i think you need two roles (one a clusterrole for the CRD, and the other this namespace scoped role for the various resources the operator needs to modify in its own namespace)


we don't want to give the samples operator cluster-wide access to all these resources.

Reply:
yeah I was on that path with my first comment, then somehow got turned around ... updating ...